Firmware/ultralcd.cpp

```diff
@@ -5930,14 +5925,7 @@ void bowden_menu() {
 					Sound_MakeSound(e_SOUND_TYPE_BlindAlert);
 				}
 
-				lcd_set_cursor(0, 0);
-				lcd_print(' ');
-				lcd_set_cursor(0, 1);
-				lcd_print(' ');
-				lcd_set_cursor(0, 2);
-				lcd_print(' ');
-				lcd_set_cursor(0, 3);
-				lcd_print(' ');
+				lcd_puts_at_P(0, 0, PSTR(" \n \n \n "));
```

Comment:
@leptun we may even extract duplicit strings into a variable and reuse them

Reply:
I would go even further -  extract code to clear first column 3 or 4 rows into separate function, as this code
`lcd_puts_at_P(0, 0, PSTR(" \n \n \n "));`
or
`lcd_puts_at_P(0, 1, PSTR(" \n \n "));`

Is repeated several times.